src/style.scss

```diff
@@ -175,17 +177,153 @@
     padding: var(--auro-size-xs) 0;
   }
 
-  .details-slot {
+  .detailsSlot {
     padding: 0;
   }
 }
 
+// =-=-=-=-=-=-=-=-=-=-= shade =-=-=-=-=-=-=-=-=-=-=-=-
+%slottedHeading {
+  margin: 0;
+  font-size: var(--auro-text-heading-400-size);
+  font-weight: var(--auro-text-body-default-weight);
+}
+
 :host([shade]) {
   @extend %chromeless;
   @extend %lowProfile;
 
+  position: relative;
+  display: block;
+
+  .details {
+    position: absolute;
+    width: 100%;
+    z-index: 1;
+    background-color: var(--auro-color-background-lighter);
+    transition: all 0.2s ease-in-out;
+  }
+
+  .trigger {
+    &::slotted(h1) {
+      @extend %slottedHeading
+    }
+
+    &::slotted(h2) {
+      @extend %slottedHeading
+    }
+
+    &::slotted(h3) {
+      @extend %slottedHeading
+    }
+
+    &::slotted(h4) {
+      @extend %slottedHeading
+    }
```

Comment:
Consider comma separated selectors here since the same CSS is applied to all 4.

```
&::slotted(h1),
&::slotted(h2),
&::slotted(h3)
&::slotted(h4) {
  @extend %slottedHeading
}
```

Reply:
It didn't work, that's why I used the `@extend` feature. I don't know why it didn't work, but ¯\_(ツ)_/¯

Reply:
So if I understand this right, this code is styles for an h1(etc) inside the trigger slot. Which doesn't appear to be working at all to me.

![Screen Shot 2021-03-04 at 9 47 22 AM](https://user-images.githubusercontent.com/23223137/110006734-d85cce00-7cce-11eb-99a5-270908df0b72.png)

Reply:
Are you sure you created a clean build of the element before running the server? 

<img width="1100" alt="Screen Shot 2021-03-05 at 4 07 14 PM" src="https://user-images.githubusercontent.com/181089/110187369-f90b4d80-7dcc-11eb-9f65-81a754da7fe9.png">

Reply:
This is the first time I have ever built and run a local copy of this repo and there are no changes on my side other than to add the h1 to the demo file. Should be as clean as it gets, Dale.

Reply:
Can you validate https://accordion-shade.surge.sh/components/auro/accordion

I am at a loss why this appears to be working for me and not you? Also, I revised the following code. Unsure why that was not working before? 

```
  .trigger {
    &::slotted(h1),
    &::slotted(h2),
    &::slotted(h3),
    &::slotted(h4) {
      margin: 0;
      font-size: var(--auro-text-heading-400-size);
      font-weight: var(--auro-text-body-default-weight);
    }
  }
 ```

Reply:
Ah I see the difference now:

In your example you are making the `<h*>` element the slot itself. In my example I replaced the `auro-icon` and text string _inside_ the slot with the `<h1>`.

If you don't want to support that maybe just add a section in the Dos and Donts about using h* attributes differently than other content that can go in the trigger slot. Or update the selector to work for both situations. Whatever works best for you.

Reply:
I updated the doc in the demo to be more specific as to uses that are supported for the trigger. 
https://accordion-shade.surge.sh/components/auro/accordion

I feel that this will satisfy the first release.